recipes/pop/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+go build -buildmode=pie -trimpath -o=${PREFIX}/bin/pop -ldflags="-s -w -X main.Version=${PKG_VERSION}"
+go-licenses save . --save_path=license-files --ignore github.com/mattn/go-localereader
```

Comment:
I believe it's now a conda-forge policy that we manually add licenses to the recipe that have to be skipped by license tools like `go-licenses` if they exist.  For this particular case, the upstream license is here: https://github.com/mattn/go-localereader/blob/master/LICENSE.  And you can see an example of how we do this in this feedstock: https://github.com/conda-forge/sttr-feedstock/tree/main/recipe.

Once that's fixed this looks good to me.

Reply:
Done!